docker/docker-compose-logging.yml

```diff
@@ -0,0 +1,155 @@
+# Usage
+#   Start:          docker compose -f docker-compose.yml -f docker-compose-logging.yml up
```

Comment:
@filipecabaco - Docker compose files are "additive" using the `-f` function

You only need Vector + Logflare in this, and you can remove any service that's already in `docker-compose.yml`

Reply:
I'm adding the other services as a way to properly update the `depends_on` properties so we can catch all logs at the right time.